### DISCUSSION TITLE
Spooky Bunkers on Venus

### DISCUSSION THREAD

**Discussion by obsidiousdev** [shun126/UE5-DungeonGeneratorDemo#38] [Show and tell]
I couldn't fit the video into an attachment :P 
The game's working title is Venusian Survivor and I've been working on it by myself; WITH the help of shun126 for creating their wonderful dungeon generator :) thank you again for making such a well made plugin!! On the other hand, I dunno if my project will ever be complete, but I have high hopes! lol
[Here is the video.](https://www.youtube.com/watch?v=wm-MeuuTo9A)
![image](https://github.com/user-attachments/assets/7d89c2d2-5ada-4f16-b6be-a00919a7a30b)
![image](https://github.com/user-attachments/assets/b4b446e3-f5f9-443f-9e65-c0ca22e6fef1)

**Comment by shun126** [comment]
Thank you for reaching out! The screenshot looks amazing! The lighting is very impressive. If you find any missing features in the `DungeonRoomSensorBase` actor, please let me know. Wishing you all the best for its completion!
